rust/cubesql/cubesql/src/compile/engine/udf.rs

```diff
@@ -266,7 +266,7 @@ type OidType = UInt32Type;
 // TODO: Combine with downcast
 fn cast_oid_arg(argument: &ArrayRef, name: &str) -> Result<ArrayRef> {
     match argument.data_type() {
-        DataType::Int32 | DataType::Int64 => {
+        DataType::Int32 | DataType::Int64 | DataType::Utf8 | DataType::LargeUtf8 => {
```

Comment:
Since we're changing this as a temporary solution, A TODO notice would be a good addition.